packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -39,15 +39,14 @@ import { SinonSpy, SinonStub, spy } from 'sinon';
 import * as vscode from 'vscode';
 import { channelService } from '../../../src/channels';
 import { BaseDeployExecutor } from '../../../src/commands';
-import {
-  DeployExecutor,
-  DeployRetrieveExecutor,
-  RetrieveExecutor
-} from '../../../src/commands/baseDeployRetrieve';
+import { DeployRetrieveExecutor } from '../../../src/commands/baseDeployRetrieve';
+import { DeployExecutor } from '../../../src/commands/DeployExecutor';
+import { RetrieveExecutor } from '../../../src/commands/RetrieveExecutor';
```

Comment:
can these three imports be combined at `../../../src/commands`? or (`../../../source/commands/executors`?)